pymesync.py

```diff
@@ -0,0 +1,60 @@
+"""
+pymesync - Python TimeSync Module
+
+Allows for interactions with the TimeSync API
+- timesync.send_time(baseurl) -- Sends time to baseurl (TimeSync)
+"""
+import errno
+import json
+import requests
+
+
+class TimeSync(object):
+
+    def __init__(self, baseurl, user, password, auth_type):
```

Comment:
What happens if one of these parameters isn't passed? Should it fail gracefully?